serverless.yml

```diff
@@ -104,6 +104,24 @@ custom:
     filterPattern: "?ERROR ?CRITICAL"
 
 functions:
+  ManageSubscriptions:
+    handler: src.subscriptions_handler.manage_subscriptions
+    role:
+      Fn::Sub:
+        - arn:aws:iam::${accountId}:role/csr-Lambda-Role
+        - accountId:
+            Ref: AWS::AccountId
+    environment:
+      AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      STAGE: ${self:provider.stage}
+      LOG_LEVEL: INFO
```

Comment:
Are we sure we want the log level set to `INFO`.

Reply:
that was for debugging, I can remove ...